### DISCUSSION TITLE
localize with .ims file

### DISCUSSION THREAD

**Discussion by YabinWang113** [jungmannlab/picasso#304] [Q&A]
When load the ims file into the localize part, click the fit, there is an error said that "MovieMapper object has no attribute use_dask". Could you please share the solution if you have the same issue?

**Comment by rafalkowalewski1** [comment]
Hi, 
could you please send me the file you mentioned?

Bests,
Rafal

**Reply by YabinWang113** [reply]
Thank you for your reply!
Could you please give me your email address, then I can send the data to you with the WeTransfer.

**Reply by rafalkowalewski1** [reply]
Yes, of course. kowalewski@biochem.mpg.de
